compiler/options.nim

```diff
@@ -124,7 +124,7 @@ type
     disabledSf, writeOnlySf, readOnlySf, v2Sf
 
   TSystemCC* = enum
-    ccNone, ccGcc, ccLLVM_Gcc, ccCLang, ccLcc, ccBcc, ccDmc, ccWcc, ccVcc,
+    ccNone, ccGcc, ccaarch64NoneElfGcc, ccLLVM_Gcc, ccCLang, ccLcc, ccBcc, ccDmc, ccWcc, ccVcc,
```

Comment:
Again I'd probably call this ``ccSwitch`` or ``ccNintendoSwitch``.

Reply:
Alright, that makes more sense. I wasn't sure what to call it, so ccNintendoSwitch shall be it's name!